quiz--project/src/home/Question.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+
+const Question = () => {
+
+
+  return (
+    <>
+    
+    </>
+  );
+};
+
+export default Question;
```

Comment:
Hope you will modify this in future, empty component rn

Reply:
yes i will